### ISSUE TITLE
Room list entries have poor colour contrast

### ISSUE THREAD

**Issue by twassman** [element-hq/element-web#32554]
### Steps to reproduce

1. Join some rooms
2. View the room list in light mode

#### What did you expect?

The room list entries should have sufficient colour contrast. (Previously, they were `#000000` on `#FFFFFF`, with a WCAG AAA compliant contrast ratio of 21:1.)

#### What happened instead?

Room list entries for rooms with no unread messages are (seemingly as of #32421) `#656D77` on `#FFFFFF`, with a contrast ratio of 5.24:1, failing WCAG AAA recommendations.

### Browser information

Firefox 148.0b3

### URL for webapp

staging.element.io

### Application version

Element version: 1.12.11-rc.1

### Will you send logs?

No

**Comment by florianduros** [comment]
The goal was to aim for WCAG AA and not WCAG AAA.
@gaelledel

**Comment by daniellekirkwood** [comment]
Thanks for raising this ticket, Accessibility remains a long-standing and core part of our product roadmap. Over the past 12 months we’ve made measurable progress towards WCAG 2.1 AA & 2.2 AA compliance. Our target is to stay focussed on this goal before striving for AAA.

We'll keep this ticket open and in our backlog so that we can easily reach for it when we hit the next phase of A11y at Element.

**Comment by gaelledel** [comment]
From a visual perspective the intention with this update is to ensure that the variety of states available are clear and distinct from each other. As we saw in previous version, it was not efficient enough to rely on text boldness alone to ensure quick differentiation between states. Using secondary text colour for read and muted states better highlights the difference with active states that are using primary colour for text. 
It was also necessary as a more holistic visual decision to increase the contrast of hover and selected states backgrounds to further emphasise what state the room cell is in. This was done in order to maximise contrast ratio, recognition and differentiation between states and as a result, increase the overall accessibility of the room list.

**Comment by TemoCrevlis** [comment]
**Thank you** for outlining the progress being made toward WCAG 2.1 AA and 2.2 AA compliance. It’s encouraging to hear that accessibility remains a core focus on the roadmap. That said, I’d like to offer some additional input in relation to this ticket.
**I use dark mode** and the variations in text I see in the Room list are: 
- White when selected
- White+Bold when unread,
- Grey otherwise.

Would it be possible to **display Room titles in a lighter shade of grey**, at least until a more technical solution is ready?
Grey text on a dark background works well for less important text, however, when quickly looking for a Room that may not have unread messages, it requires slightly more focus.

With message previews being the same grey colour as the title (read Rooms), And the listing being relatively compact - making the Room title a lighter shade would significantly improve scannability.

I appreciate you keeping the ticket open and possibly considering this feedback as part of future updates.
